src/valid.js

```diff
@@ -201,7 +201,8 @@ exports.isArg = function(node) {
 
 // Test to see if a node is a passable mithril invocation
 exports.isMithril = function(node) {
-    var first = node.arguments[0];
+    var first = node.arguments[0],
+        argumentsLength = node.arguments.length;
```

Comment:
Don't bother caching this, it's more confusing than just referencing it directly.